### PR TITLE
Adds Mexican tourists to the restaurant system, plus some QOL tweaks.

### PR DESCRIPTION
**Pull request by ArcaneMusic**
## About The Pull Request

Alright, brass tacks, adds new Mexican tourists to the restaurant. Their orders are fairly traditional spanish and mexican food, their dialog sound is some pretty classy spanish guitar, and by god they care about football. REAL football.

This also created a few QOL tweaks that I noticed and changed along the way:

- Chef Holosign creator for restaurant seating didn't look very unique due to using the standard holosign sprite, so I gave it a nice recolor with a bright white and red restaurant pattern on an edge to help it stand out a bit.
- The two taco recipies have been given a minor name tweak, which actually already existed in the crafting menu, but didn't change the names of the two tacos, so now the actual food tacos are classic tacos ( Cheese, meat, cabbage, tortilla) and plain tacos (tortilla, cheese, meat).

Mexibot
_The robot sprite._

## Why It's Good For The Game

Adds more variety to the restaurant system, keeps it tasteful in flavor and the foods themselves aren't cooked very often on station, so this helps bump up food variety on the whole.
Helps to distinguish the existing food on station a bit with the variable name, and helps to make the venue specific aspects of the restaurant system stand out a tiny bit more.

## Changelog
:cl:
add: There are now space tourists from south of the border. Andele!
qol: The two taco recipes are now distinguished by name in game to help tell the two apart.
imageadd: The Restaurant Holoseat Creator now uses a slightly different sprite to help it stand out.
/:cl:

Demo Video (Note: this was the exact moment I realized I needed to change the name):
https://discord.com/channels/326822144233439242/326831214667235328/817188033446412290

### CODE REVIEW

**Comment by tralezab** [comment]
damn NICE

File: code/modules/food_and_drinks/restaurant/customers/_customer.dm [hunk 1]
```diff
@@ -125,3 +125,20 @@
...
+	base_icon = "mexican"
+	prefix_file = "strings/names/mexican_prefix.txt"
+	speech_sound = 'sound/creatures/tourist/tourist_talk_mexican.ogg'
+	clothing_sets = list("mexican_poncho")
+	orderable_objects = list(
+	/datum/venue/restaurant = list(/obj/item/food/taco/plain = 25, /obj/item/food/taco = 15 , /obj/item/food/burrito = 15, /obj/item/food/fuegoburrito = 1, /obj/item/food/cheesyburrito = 4, /obj/item/food/nachos = 10, /obj/item/food/cheesynachos = 6, /obj/item/food/pie/dulcedebatata = 2, /obj/item/food/cubannachos = 3, /obj/item/food/stuffedlegion = 1),
+	/datum/venue/bar = list(/datum/reagent/consumable/ethanol/whiskey = 6, /datum/reagent/consumable/ethanol/tequila = 20, /datum/reagent/consumable/ethanol/tequila_sunrise = 1, /datum/reagent/consumable/ethanol/beer = 15, /datum/reagent/consumable/ethanol/patron = 5, /datum/reagent/consumable/ethanol/brave_bull = 5, /datum/reagent/consumable/ethanol/margarita = 8))
+
+
+	found_seat_lines = list("Como te va, space station 13?", "Who's ready to party!", "Ah, muchos gracias.", "Ahhh, smells like mi abuela's cooking!")
```
Comment (Ghommie) [inline code/modules/food_and_drinks/restaurant/customers/_customer.dm:140]:
```suggestion
	found_seat_lines = list("¿Como te va, space station 13?", "Who's ready to party!", "Ah, muchas gracias.", "Ahhh, smells like mi abuela's cooking!")
```

File: code/modules/food_and_drinks/restaurant/customers/_customer.dm [hunk 2]
```diff
@@ -125,3 +125,20 @@
...
+	cant_find_seat_lines = list("En Serio? Seriously, no seats?", "Andele! I want a table to watch the football match!", "Ay Caramba...")
```
Comment (Ghommie) [inline code/modules/food_and_drinks/restaurant/customers/_customer.dm:141]:
```suggestion
	cant_find_seat_lines = list("¿En Serio? Seriously, no seats?", "Andele! I want a table to watch the football match!", "Ay Caramba...")
```

File: strings/names/mexican_prefix.txt [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+Bandito
+Mariachi
+Presidente
+Embajador
+Redactor
+Deportista
+Gonzales
+Rio
+Pele
+Conqustador
```
Comment (Ghommie) [inline strings/names/mexican_prefix.txt:12]: That's a typo.
```suggestion
Conquistador
```

File: code/modules/food_and_drinks/restaurant/customers/_customer.dm [hunk 4]
```diff
@@ -125,3 +125,20 @@
...
+	leave_mad_lines = list("Aye dios mio, I'm out of here.", "Esto es ridiculo! I'm leaving!", "I've seen better cooking at taco campana!", "I though this was a restaurant, pero es porqueria!")
```
Comment (Ghommie) [inline code/modules/food_and_drinks/restaurant/customers/_customer.dm:142]:
```suggestion
	leave_mad_lines = list("Aye dios mio, I'm out of here.", "Esto es ridículo! I'm leaving!", "I've seen better cooking at taco campana!", "I though this was a restaurant, pero es porquería!")
```

File: strings/names/mexican_prefix.txt [hunk 5]
```diff
@@ -0,0 +1,13 @@
+Ranchero
+Senor
```
Comment (Ghommie) [inline strings/names/mexican_prefix.txt:2]:
```suggestion
Señor
```

**Review by Ghommie — CHANGES_REQUESTED** [review]
Bettering this hollywood spanish a little.

**Comment by Pugie** [comment]
Sure, boss, this sounds good.

**Comment by maxymax13** [comment]
all these restaurant costumer prs never give any love to the bartender part

**Comment by ArcaneMusic** [comment]
> all these restaurant costumer prs never give that much love to the bartender part

Any suggestions to keep it more neutral? Admittedly I probably lean closer towards food than drinks here.

**Comment by tralezab** [comment]
I think that's something qust is gonna have to address at some time in the future in general, every single customer is like that

**Review by Twaticus — APPROVED** [review]

**Pull request merged**